from .passport_element_error_source import PassportElementErrorSource

### FILE IS AUTOGENERATED. DO NO EDIT IT BY HAND ###

class PassportElementErrorSourceDataField(PassportElementErrorSource):
    """
    One of the data fields contains an error. The error will be considered resolved when the value of the field changes

    :param field_name: Field name
    """
    __slots__ = ("field_name", "_extra", "_client_id", "_type")

    def __init__(self, field_name: str = ""):
        self.field_name = field_name
        self._type = "passportElementErrorSourceDataField"